File: Daily/00058. Length of Last Word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let trimmedString = s.trim(); 
    let arr = trimmedString.split(' ');
    let lastWord = arr[arr.length - 1];
    return lastWord.length;
};

File: LeetCode Daily/00058. Length of Last Word.go
func lengthOfLastWord(s string) int {
	var res int
	for i := len(s) - 1; i >= 0; i-- {
		if s[i] != ' ' {
			res++
		} else if res > 0 {
			return res
		}
	}
	return res
}

